pkg/storage/fs/local/local.go

```diff
@@ -257,6 +257,11 @@ func (fs *localfs) isShareFolderChild(ctx context.Context, p string) bool {
 
 func (fs *localfs) normalize(ctx context.Context, fi os.FileInfo, fn string) *provider.ResourceInfo {
 	fn = fs.unwrap(ctx, path.Join("/", fn))
+	owner, err := getUser(ctx)
```

Comment:
This will add the current logged in user, I'm not sure this will apply for shared resources

Reply:
I think we should rely on what the storage gives us, for example, EOS wll always tell us the owner

Reply:
So I thnk in the case of local storage is okay to use it like that.

Reply:
Yes, in that case, I thought we might as well return the "owner of the reference" since the reference doesn't have any info about the owner of the original file. Maybe we can pass this info about the original owner to the CreateReference method and then save that as metadata along with the reference (currently we store only the target).

Reply:
IMHO we should really get the owner, also for local storage, independent from the currently logged in user. Or did I misunderstand @labkode's comment?

Reply:
@glpatcern yes, in case of share references, we don't have any info about the original owner in this context. That's why I suggested passing it along when the reference is created. We'll need to add this field to the cs3apis protobuf.
For cases other than the references, the logged-in user will always be the owner.